Game/lv/Game.po

```diff
@@ -1082,7 +1082,7 @@ msgstr "...par tiem jāspriež citu dienu."
 #: /Game/Resources/Localisation/ST_Recordings.ST_Recordings
 msgctxt "Recordings,LiamJonesDiary1Part1"
 msgid "Liam Jones (recording): This is the uh, Liam Jones, audio diary, #1, uh"
-msgstr "Laiams Džonss (ieraksts): Šī ir, ē, Laiama Džonsa audio dienasgrāmata #1, ē"
+msgstr "Liams Džonss (ieraksts): Šī ir, ē, Liama Džonsa audio dienasgrāmata #1, ē"
```

Comment:
Forgive the stupid questions - what's the difference between Liams Džonss and Liama Džonsa? Is the former the version you use when attributing speech, and the latter the form you use when referring to someone within speech? (I realise this is a feature of the original translation too)

Reply:
Exactly! `Liams` is called nominative, `Liama` is called genitive. These two among others are not found in english.